package.sh

```diff
@@ -95,12 +95,14 @@ chmod u+x package/heroku/*.sh
 
 # TKG
 
-cp -r deployment/tkg package
-cp -r deployment/helm/postfacto-*.tgz package/tkg
-cp -r deployment/deploy-tkg.sh package/tkg/deploy.sh
-cp -r deployment/upgrade-tkg.sh package/tkg/upgrade.sh
-cp -r deployment/mixpanel.sh package/tkg/mixpanel.sh
-chmod u+x package/tkg/*.sh
+if [ -d deployment/helm/postfacto-*.tgz ]; then
```

Comment:
It seems like `package.sh` requires extra steps to create assets for TGZ deployment. I tried mimicking the steps that Github Actions performs, but it seems like that is no longer working (I run into problems at "Helm build"). 

I propose that we let packaging work without TGZ, until `package.sh` can perform can take care of it without extra steps.